tests/drop_counters/test_drop_counters.py

```diff
@@ -161,6 +161,34 @@ def enable_counters(duthost):
                 duthost.command("counterpoll {} disable".format(port))
 
 
+@pytest.fixture
+def mtu_setup(duthost):
+    """ Fixture which prepare port MTU configuration for 'test_ip_pkt_with_exceeded_mtu' test case """
+    class MTUConfig(object):
+        iface = None
+        @classmethod
+        def set_mtu(cls, mtu, iface):
+            if "PortChannel" in iface:
+                res = duthost.command("redis-cli -n 4 hset \"PORTCHANNEL|{}\" mtu {}".format(iface, mtu))["stdout"]
+            elif "Ethernet" in iface:
+                res = duthost.command("redis-cli -n 4 hset \"PORT|{}\" mtu {}".format(iface, mtu))["stdout"]
+            if res != "0":
+                pytest.fail("Unable to change MTU on interface {}".format(iface))
+            cls.iface = iface
+        @classmethod
+        def restore_mtu(cls, mtu=9100):
+            if "PortChannel" in cls.iface:
+                res = duthost.command("redis-cli -n 4 hset \"PORTCHANNEL|{}\" mtu {}".format(cls.iface, mtu))["stdout"]
+            elif "Ethernet" in cls.iface:
+                res = duthost.command("redis-cli -n 4 hset \"PORT|{}\" mtu {}".format(cls.iface, mtu))["stdout"]
+            if res != "0":
+                pytest.fail("Unable to change MTU on interface {}".format(cls.iface))
+    try:
+        yield MTUConfig
+    finally:
+        MTUConfig.restore_mtu()
```

Comment:
better to read it, save it and restore saved

Reply:
Agree, but in such case should be stored MTU from all ports and restored as well.
In current implementation MTU is only changing on one port.